src/Mado/QueryBundle/Tests/Services/StringParserTest.php

```diff
@@ -39,7 +39,7 @@ public function testSplitStringInTokenViaUnderscore(
      * @covers Mado\QueryBundle\Services\StringParser::tokenize
      * @dataProvider tokens
      */
-    public function testTo(
+    public function testProvideTokenAtPosition(
```

Comment:
@AlessandroMinoccheri, is it better `testProvideTokenAtPosition` or `testProvideTokenAtIndex`?

Reply:
I prefer testProvideTokenAtIndex but is not so different I think

Reply:
I'll leave testProvideTokenAtPosition

Reply:
ok!